test/api/locks/unlock_door.test.ts

```diff
@@ -0,0 +1,77 @@
+import test from "ava"
+
+import { getTestServer } from "fixtures/get-test-server.ts"
+
+test("POST /locks/unlock_door", async (t) => {
+  const { axios, db, seed } = await getTestServer(t)
+
+  const device = db.devices.find(
+    (d) => d.workspace_id === seed.ws2.workspace_id
+  )
+
+  if (device == null) {
+    t.fail("Expected at least one device in ws2")
+    return
+  }
```

Comment:
```suggestion
  t.assert(device, "Expected at least one device in ws2")
```

Suggestion applies to the other instances of this pattern I see here.